### ISSUE TITLE
Pseudo code in ruby problem solving lesson is formatted poorly

### ISSUE THREAD

**Issue by ChargrilledChook** [TheOdinProject/curriculum#18528]
The pseudo code examples in this lesson appear as blocks of text on the website and are hard to read eg

![odin-site-pcode1](https://user-images.githubusercontent.com/58506115/85695022-fffc9f00-b71a-11ea-83d0-1524e092a5e5.png)

![odin-site-pcode2](https://user-images.githubusercontent.com/58506115/85695045-0428bc80-b71b-11ea-8f2f-fe971de0590a.png)
***
However the same examples look good on the github page eg

![odin-repo-pcode1](https://user-images.githubusercontent.com/58506115/85695267-3803e200-b71b-11ea-996e-e07177e202cf.png)

![odin-repo-pcode2](https://user-images.githubusercontent.com/58506115/85695278-3afed280-b71b-11ea-9b34-35375b166c55.png)
***
These examples would be much more effective / easy to read if the live site looked like the examples from the repo.

[Live page](https://theodinproject.com/courses/ruby-programming/lessons/problem-solving)

[On Github](https://github.com/TheOdinProject/curriculum/edit/master/ruby_programming/basic_ruby/problem_solving.md)

**Comment by dm-murphy** [comment]
@marvingay I noticed this issue after fixing the backticks for this page. I can change this as well if that's alright. I think separating each pseudo code line onto its own line might be sufficient and more readable. That way it also keeps it distinguished from the full code blocks.

**Comment by marvingay** [comment]
@dm-murphy  Go for it.

**Comment by dm-murphy** [comment]
@marvingay @ChargrilledChook  

I believe I have resolved this issue. The 3 backticks for the pseudo code blocks were replaced with 3 tildes and that separated the lines. I have also submitted the same fix to the JavaScript Problem Solving page along with the rest of this page's editorial fixes so once that PR is merged the pseudo code blocks should have line spacing on both pages. 

It should be noted the text color is now black, not magenta. Don't know if that is an issue for anyone but it appears readable on my end and is a different font than the rest of the body text to distinguish the pseudo code from the paragraphs.
